packages/stark-core/src/common/environment/environment.intf.ts

```diff
@@ -0,0 +1,38 @@
+import { NgModuleRef } from "@angular/core";
+
+/**
+ * Interface to be implemented by all environments
+ */
+export interface StarkEnvironment {
+	/**
+	 * Whether the current environment is production
+	 */
+	production: boolean;
```

Comment:
As discussed, we can probably drop those properties in favor of the global/ambient ones which are maybe easier to use

Reply:
Ok, I've updated this PR with the changes wde discussed.

@SuperITMan @dsebastien Feel free to review again and merge if everything is ok